### ISSUE TITLE
`skywire cli proxy start` not working as expected v1.3.20

### ISSUE THREAD

**Issue by 0pcom** [skycoin/skywire#1779]
```
[2024-03-20T10:44:06.503768555-05:00] DEBUG (STDOUT) [proc:skysocks-client:478496d5502c46c59fba81ee1670026a]: unknown flag: --srv

```

but the flag IS THERE

```
$ skywire app skysocks-client --help

  ┌─┐┬┌─┬ ┬┌─┐┌─┐┌─┐┬┌─┌─┐   ┌─┐┬  ┬┌─┐┌┐┌┌┬┐
  └─┐├┴┐└┬┘└─┐│ ││  ├┴┐└─┐───│  │  │├┤ │││ │
  └─┘┴ ┴ ┴ └─┘└─┘└─┘┴ ┴└─┘   └─┘┴─┘┴└─┘┘└┘ ┴



Flags:
      --addr string   Client address to listen on (default ":1080")
      --http string   http proxy mode
      --srv string    PubKey of the server to connect to
  -v, --version       version for skysocks-client

```

If I change the autostart in the config to true

```
			{
				"name": "skysocks-client",
				"binary": "skywire",
				"args": [
					"app",
					"skysocks-client",
					"--addr",
					":1080",
					"--srv",
					"031952d79eca8852a7c9fc2ae24ff16ddfe00665f62494845382fa53a8dab4ac4b"
				],
				"auto_start": true,
				"port": 13
			},

```

and restart the visor ; THEN IT WORKS. Why???

**Comment by 0pcom** [comment]
```
$ skywire-cli proxy stop --name skysocks-client
^C^C^C^C^C^C^C^C

```

proxy stop isn't working as well

**Comment by 0pcom** [comment]
fixed #1781
